lib/matplotlib/backends/backend_svg.py

```diff
@@ -1025,18 +1025,6 @@ def _adjust_char_id(self, char_id):
         return char_id.replace("%20", "_")
 
     def _draw_text_as_path(self, gc, x, y, s, prop, angle, ismath, mtext=None):
-        """
```

Comment:
Is this because the docstring is identical to the parent? If so, please add a comment line
```
# docstring inherited
```
here.

Reply:
Done!